### PR TITLE
Feature/tailwind blog listing

### PR DESCRIPTION
**Pull request by clari182**
- Part of issue #604 regarding "Port the blog listing (https://incidentdatabase.ai/blog) to use Tailwind in the main content area"

This feature also includes logic to concurrently use both bootstrap and tailwind.
Since tailwind is meant to be executed at the end, overwriting previous CSS stylings, it was showing incompatibilities with the current bootstrap implementation because Tailwind automatically loads a default theme with default stylings for the body, button, and more elements. 
This is why global.css is loaded first in gatsby-browser.js and bootstrap later.
Until we eliminate bootstrap, we need to be careful when implementing stylings and always use prefix `tw-` in tailwind classes.
For example if we want a div to be flex, we should add the class `tw-flex` 

Deploy preview: https://deploy-preview-825--staging-aiid.netlify.app/

### CODE REVIEW

**Comment by cesarvarela** [comment]
Please fix conflicts

**Comment by clari182** [comment]
@cesarvarela conflicts fixed

**Review by cesarvarela — APPROVED** [review]
Looks good should be merged after #822

**Review by cesarvarela — CHANGES_REQUESTED** [review]
There is a weird margin on the blog listing:

image


The links are black:

image

**Comment by cesarvarela** [comment]
please fix conflicts

**Review by cesarvarela — APPROVED** [review]

File: site/gatsby-site/src/components/blog/PostPreview.js [hunk 1]
```diff
@@ -1,47 +1,42 @@
...
-          </Card.Body>
-        </Col>
-      </Row>
+        </Card.Title>
+        <div className="tw-font-medium">
+          {format(new Date(post.frontmatter.date), 'MMM d, yyyy')}
+        </div>
+        <div>
+          {' '}
+          {post.excerpt}... <LocalizedLink to={post.frontmatter.slug}>(Read More)</LocalizedLink>
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/blog/PostPreview.js:32]: Missing `Trans`

**Review by cesarvarela — CHANGES_REQUESTED** [review]
There is one text missing translations

**Comment by clari182** [comment]
@cesarvarela missing translation added, margins and links fixed

**Review by cesarvarela — APPROVED** [review]

**Pull request merged**